app/src/main/java/net/skyscanner/backpack/demo/MainActivity.kt

```diff
@@ -52,66 +41,23 @@ import net.skyscanner.backpack.demo.data.RegistryItem
  */
 class MainActivity : BpkBaseActivity() {
 
+  @OptIn(ExperimentalAnimationApi::class, ExperimentalMaterialNavigationApi::class)
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
     setContent {
-      BackpackDemoTheme {
-        ComponentScreen()
-      }
-    }
-  }
-
-  @Composable
-  private fun ComponentScreen(modifier: Modifier = Modifier) {
-    val state = rememberTopAppBarState()
-    Column(modifier = modifier.nestedScroll(state)) {
-      val context = LocalContext.current
-      BpkTopNavBar(
-        state = state,
-        navIcon = NavIcon.None,
-        title = stringResource(R.string.app_name),
-        actions = listOf(
-          IconAction(
-            icon = BpkIcon.Settings,
-            contentDescription = stringResource(R.string.settings_title),
-            onClick = {
-              val intent = Intent(context, SettingsActivity::class.java)
-              context.startActivity(intent)
-            },
+      DemoScaffold {
+        DestinationsNavHost(
```

Comment:
I used the library, which uses a KSP processor to generate the boilerplate needed for Compose Navigation library.
This library also supports Accompantist Navigation Animation (https://google.github.io/accompanist/navigation-animation/), which is a fork of official library with custom nav animations support.